sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/ThroughputControlMode.java

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+public enum ThroughputControlMode {
```

Comment:
We can move this enum into the implementation package if my suggestion regarding name the method such that you don't need the extra parameters is acceptable. The "get" method will become a boolean one, some along the lines of "isLocallyControlledMode" or "isDistributedControlledMode", which you can use to toggle between the different modes

Reply:
Yup, as discussed in the team meeting, ThroughputControlMode will be moved to implementation.
and we are going to use the following pattern;

container.enableThroughputLocalControlGroup(...)